### PR TITLE
Career page

### PR DESCRIPTION
**Pull request by darmhoo**

### CODE REVIEW

File: src/components/CareerPage.vue [hunk 1]
```diff
@@ -0,0 +1,311 @@
...
+				fileSelected: null,
+				event: null,
+				config: {
+					bucketName: 'altara-staging',
+					dirName: 'media',
+					region: 'us-east-1',
+					accessKeyId: process.env.VUE_APP_AWS_ACCESS_KEY,
+					secretAccessKey: process.env.VUE_APP_AWS_SECRET_KEY,
+					s3Url: ' https://altara-staging.s3.amazonaws.com/',
+				},
```
Comment (teebarg) [inline src/components/CareerPage.vue:213]: can we have this as env variable?

File: src/components/CareerPage.vue [hunk 2]
```diff
@@ -0,0 +1,315 @@
...
+					let result = await S3CustomClient.uploadFile(
+						this.fileSelected,
+						this.fileSelected.type,
+						undefined,
+						this.fileSelected.name,
+						undefined
+					);
+					return result.location;
+				} catch (error) {
+					console.log(error);
```
Comment (teebarg) [inline src/components/CareerPage.vue:233]: remove log

File: src/components/CareerPage.vue [hunk 3]
```diff
@@ -0,0 +1,315 @@
...
+				} finally {
+					console.log('done');
```
Comment (teebarg) [inline src/components/CareerPage.vue:235]: remove log

File: src/components/CareerPage.vue [hunk 4]
```diff
@@ -0,0 +1,315 @@
...
+				roles: '',
+				location: '',
+				resume: '',
+				form_sent: false,
+				fileSelected: null,
+				event: null,
+				config: {
+					bucketName: 'altara-staging',
+					dirName: 'media',
+					region: 'us-east-1',
```
Comment (teebarg) [inline src/components/CareerPage.vue:209]: let's create an env variable for region as well

File: src/components/CareerPage.vue [hunk 5]
```diff
@@ -0,0 +1,315 @@
...
+				email_address: '',
+				phone_number: '',
+				roles: '',
+				location: '',
+				resume: '',
+				form_sent: false,
+				fileSelected: null,
+				event: null,
+				config: {
+					bucketName: 'altara-staging',
```
Comment (teebarg) [inline src/components/CareerPage.vue:207]: let's put bucketName in a variable

File: src/components/CareerPage.vue [hunk 6]
```diff
@@ -0,0 +1,315 @@
...
+					dirName: 'media',
```
Comment (teebarg) [inline src/components/CareerPage.vue:208]: change upload folder to `cv-upload`

File: src/components/CareerPage.vue [hunk 7]
```diff
@@ -0,0 +1,315 @@
...
+
+	export default {
+		components: {
+			Modal,
+			loaderVue,
+		},
+		data() {
+			return {
+				loader: null,
+				formURL: process.env.VUE_APP_URL_JOBFORM_STAGING,
```
Comment (teebarg) [inline src/components/CareerPage.vue:196]: what is this for?

Comment (darmhoo) [reply src/components/CareerPage.vue]: the google sheet url

File: src/components/CareerPage.vue [hunk 8]
```diff
@@ -0,0 +1,313 @@
...
+              <label for="phone_number" class="mb-0.5 ml-2 text-sm"
+                >Roles you are interested in: *</label
+              >
+              <select
+                name="roles"
+                v-model="roles"
+                class="mt-1 block w-full rounded-md border focus:outline-none border-gray-300 shadow-sm focus:border-indigo-300 focus:ring focus:ring-indigo-200 focus:ring-opacity-50 h-10 text-sm shadow-sm px-2 py-1"
+              >
+                <option value="default" class disabled>Roles</option>
+                <option class="text-sm" value="Direct Sales Agent">
```
[the remark this answers is not in the document]
Comment (teebarg) [reply src/components/CareerPage.vue:100]: @darmhoo FYA

File: src/components/CareerPage.vue [hunk 9]
```diff
@@ -0,0 +1,340 @@
...
+              >
+
+              <select
+                name="location"
+                v-model="location"
+                @change="validate()"
+                class="mt-1 block w-full rounded-md border focus:outline-none border-gray-300 shadow-sm focus:border-indigo-300 focus:ring focus:ring-indigo-200 focus:ring-opacity-50 h-10 text-sm shadow-sm px-2 py-1"
+              >
+                <option value="default" class disabled>Area</option>
+                <option class="text-sm" value="Apata">Apata, Ibadan</option>
```
Comment (teebarg) [inline src/components/CareerPage.vue:127]: Lets put these options in an array

File: src/components/CareerPage.vue [hunk 10]
```diff
@@ -0,0 +1,340 @@
...
+      }
+      const api = new Apiservice();
+      const data = {
+        full_name: this.full_name,
+        email_address: this.email_address,
+        phone_number: this.phone_number,
+        roles: this.roles,
+        location: this.location,
+        date: new Date().toLocaleString(),
+        resume: 'https://altara-staging.s3.amazonaws.com/'+encodeURIComponent(this.resume),
```
Comment (teebarg) [inline src/components/CareerPage.vue:293]: What happens when we are live when you're hardcoding staging url here?

**Review by teebarg — APPROVED** [review]

**Pull request merged**